ci/toolchain_manager.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# Builds and publishes kubesaw components to a quay repository.  Used as a part
+# of the CI process for testing in openshift-ci.
+
+set -e
+
+# parse input
+BRANCH=${BRANCH:-pubviewer-mvp}
+QUAY_NAMESPACE=${QUAY_NAMESPACE:-konflux-workspaces}
+TAG=${2:-e2etest}
+KUBECLI=${KUBECLI:-kubectl}
+
+HOST_OPERATOR_REPO=${HOST_OPERATOR_REPO:-https://github.com/filariow/host-operator.git}
+MEMBER_OPERATOR_REPO=${MEMBER_OPERATOR_REPO:-https://github.com/codeready-toolchain/member-operator.git}
+TOOLCHAIN_E2E_REPO=${TOOLCHAIN_E2E_REPO:-https://github.com/sadlerap/toolchain-e2e.git}
+REGISTRATION_SERVICE_REPO=${REGISTRATION_SERVICE_REPO:-https://github.com/filariow/registration-service}
+
+function clone {
+    git clone --depth 2 --branch "${2}" "${1}"
+}
+
+# build & publish toolchain components
+function publish {
+    clone "${MEMBER_OPERATOR_REPO}"      "master"
+    clone "${HOST_OPERATOR_REPO}"        "${BRANCH}"
+    clone "${TOOLCHAIN_E2E_REPO}"        "${BRANCH}"
+    clone "${REGISTRATION_SERVICE_REPO}" "${BRANCH}"
```

Comment:
instead of using same branch for all add the ability to have independent branches that can override a default.

Reply:
Fixed in ded2cf3.

Reply:
OK